fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseWriterSelector.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web;
+
+
+/**
+ * Response writer selector object.
```

Comment:
Again, this should provide a bit more insight into what this exists, and why it should be used. Maybe something like:
```
Typically, a Fili program's response writing infrastructure will consist of a collection of unrelated 
ResponseWriters each responsible for serializing the ResultSet into a specific format, and a single 
ResponseWriter that chooses the correct one based on the current state of the DataApiRequest.

This interface allows customers provide a clean interface to the logic that makes that selection. Its 
sole purpose is to take a `DataApiRequest` and return the `ResponseWriter` that should be used 
to  write the response. 
```